theano/gpuarray/tests/test_reduction.py

```
@@ -14,7 +14,7 @@
 import math
 
 # Number of values to be used in test tensors (except with 0-D tensors!).
-test_size = 10000000
+test_size = 10000
```

We need to find the good fix. This just hide the error.